### PR TITLE
Add about Ashutosh

### PR DESCRIPTION
**Pull request by cyberashutosh**
please approve

### CODE REVIEW

**Review by Tanvi2698 — COMMENTED** [review]
add full stop 
after smile: and change your experience description

**Review by ghost — COMMENTED** [review]
last line still not removed

File: submissions/cyberashutosh/about.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# About Me
+My name is Ashutosh Mishra. I am an IT student at Panjab University, India.
+# Experience
+IT first year student. Just learnt about git.
```
Comment (sagar-kalra) [inline submissions/cyberashutosh/about.md]: Remove the full stop after student

**Review by sagar-kalra — CHANGES_REQUESTED** [review]

File: submissions/cyberashutosh/about.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+IT first year student. Just learnt about git. 
+# Interests
+ Making projects related to robotics and learning about them. 
+# Contact Info
+Email: [ashutosh.mishra832@gmail.com](mailto:ashutosh.mishra832@gmail.com)   
+Facebook profile: [Ashutosh Mishra](https://www.facebook.com/ashutosh.mishra)    
+Github: [cyberashutosh](https://github.com/cyberashutosh)
+# Emoji
+My favorite emoji is :smile. 
+
```
Comment (ankurbhatia24) [inline submissions/cyberashutosh/about.md]: Remove empty line.

**Review by ankurbhatia24 — CHANGES_REQUESTED** [review]
Squash your commits into one.

**Review by ankurbhatia24 — APPROVED** [review]

**Review by harryjas — APPROVED** [review]

**Pull request merged**